channeldb/db.go

```diff
@@ -9,16 +9,41 @@ import (
 	"sync"
 
 	"github.com/boltdb/bolt"
+	"github.com/go-errors/errors"
 	"github.com/roasbeef/btcd/btcec"
 	"github.com/roasbeef/btcd/chaincfg"
 	"github.com/roasbeef/btcd/wire"
 )
 
 const (
-	dbName = "channel.db"
+	dbName           = "channel.db"
+	dbFilePermission = 0600
+
+	// dbCorruptedPrefix is used for renaming the default name of the
+	// database if migration is failed.
+	dbCorruptedPrefix = "corrupted."
+
+	// dbBackupPrefix is used for constructing the name of the backup
+	// file of the database during migration process.
+	dbBackupPrefix = "backup."
 )
 
+// Migration is a function which changes the data accordingly to maintain the
```

Comment:
Also, I'm not sure this type needs to be externally exported. Database migrations should essentially be silent and obscured from the view outside of this package.